### PR TITLE
Improve service runner

### LINKED ISSUE #402: Improve service tests runner
#396 introduced a new runner, but with a small problem. The configuration file defines what runner should be used with what arguments. Those arguments were meant to be the general configuration parameters, not CLI flags (although that's true for all current runners). Therefore, the ExcludeArgs configuration option does not make much sense in that context.

Let's remove that option, implement handling of a leading `!` ourselves, and clarify that `jstest` and `service` runners support it (as well as other globs).

### PR DESCRIPTION
**Pull request by b1ron**
Closes #402.

### CODE REVIEW

**Review by AlekSi — CHANGES_REQUESTED** [review]
Let's not use github.com/mattn/go-zglob and extend the scope of the issue.

**Comment by b1ron** [comment]
The linter fails on [CI](https://github.com/FerretDB/dance/actions/runs/4870982774/jobs/8687388250?pr=413) but passes locally.

**Review by w84thesun — APPROVED** [review]
Nice one!

File: internal/jstest/jstest.go [hunk 1]
```diff
@@ -29,28 +29,46 @@ import (
...
 	_ = ctx
 
 	filesM := make(map[string]struct{})
 
+	excludeFilesM := make(map[string]struct{})
+
 	// remove duplicates if globs match same files
 	for _, f := range args {
+		if strings.HasPrefix(f, "!") {
+			f = strings.TrimPrefix(f, "!")
```
Comment (AlekSi) [inline internal/jstest/jstest.go:43]: There is a new `CutPrefix` that does both operation

**Review by AlekSi — APPROVED** [review]

File: internal/jstest/jstest.go (hunk 1 above)
Comment (b1ron) [reply internal/jstest/jstest.go]: Crisp!

**Comment by AlekSi** [comment]
You do; that looks like a GitHub bug

**Review by AlekSi — APPROVED** [review]

**Pull request merged**